src/Client.php

```diff
@@ -30,6 +31,17 @@ public function __construct($uri, $options, $driverOptions)
     }
 
     /**
+     * Drop a database.
+     *
+     * @param string $databaseName
+     * @return Result
+     */
+    public function dropDatabase($databaseName)
```

Comment:
Should the Database object have __toString() method and this code explicitly do (string)$databaseName ?